go.mod

```diff
@@ -22,7 +22,8 @@ require (
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.4.0
 	github.com/stretchr/testify v1.6.1
-	golang.org/x/tools v0.0.0-20200403190813-44a64ad78b9b
+	golang.org/x/lint v0.0.0-20200302205851-738671d3881b // indirect
+	golang.org/x/tools v0.0.0-20201019175715-b894a3290fff
```

Comment:
This lines appear to be caused by running `make bindata` and should be reverted.

Reply:
It is the go.mod of the project.
See that it still as it is and passing in the CI after we revert the changes made via the `make fix` on the bindata.

Reply:
However, I reverted that I executed go mod tidy.